models/errors-1.0.0.yaml

```diff
@@ -0,0 +1,43 @@
+    Errors:
```

Comment:
I can see in models we have `error-1.0.0.yaml` . Do we need keep both? OR delete/change old one?